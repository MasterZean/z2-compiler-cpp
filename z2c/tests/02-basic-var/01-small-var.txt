val a1 = 0s8
val a2 = -1s8
val a3 = 1s8
val a4 = -100s8
val a5 = 100s8
val a6 = -127s8
val a7 = 127s8
val a8 = -128s8