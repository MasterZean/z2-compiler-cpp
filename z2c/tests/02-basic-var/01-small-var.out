int8 a1 = 0;
int8 a2 = -1;
int8 a3 = 1;
int8 a4 = -100;
int8 a5 = 100;
int8 a6 = -127;
int8 a7 = 127;
int8 a8 = -128;
